commands/utils/params.go

```diff
@@ -233,29 +236,54 @@ func extractEnvParams() (*coreconfig.ServerDetails, Git, error) {
 	return &server, gitParams, err
 }
 
-func ReadConfig(configFilePath string) (*FrogbotConfigAggregator, error) {
-	filePath, err := filepath.Abs(configFilePath)
-	if err != nil {
-		return nil, err
+func ReadConfig(configFilePaths ...string) (config *FrogbotConfigAggregator, err error) {
+	if configFilePaths == nil {
+		configFilePaths = managementRepoPath
 	}
-	fileExist, err := fileutils.IsFileExists(filePath, false)
-	if !fileExist || err != nil {
-		// If the WD directory is not ./frogbot, look in parent directories for ./jfrog/frogbot-config.yml.
-		if filePath, err = utils.FindFileInDirAndParents(filePath, configRelativePath); err != nil {
-			return nil, &ErrMissingConfig{
-				fmt.Sprintf("%s wasn't found in the Frogbot directory and its subdirectories. Continuing with environment variables", FrogbotConfigFile),
-			}
+	// Find if the frogbot-config file is in one of the possible paths in the configFilePath array.
+	var filePath string
+	var configFileFound bool
+	for _, possibleConfigPath := range configFilePaths {
+		configFileFound, err = searchConfigFile(possibleConfigPath, &filePath)
+		_, missingConfigErr := err.(*ErrMissingConfig)
+		if err != nil && !missingConfigErr {
+			return nil, err
+		}
+		if configFileFound {
+			break
 		}
-		filePath = filepath.Join(filePath, configFilePath)
+	}
+
+	if !configFileFound {
+		return nil, &ErrMissingConfig{errConfigNotFound}
 	}
 
 	configFile, err := os.ReadFile(filePath)
 	if err != nil {
 		return nil, err
 	}
 
-	var config FrogbotConfigAggregator
-	return &config, yaml.Unmarshal(configFile, &config)
+	return config, yaml.Unmarshal(configFile, &config)
+}
+
+func searchConfigFile(possibleConfigPath string, filePath *string) (bool, error) {
+	fullPossiblePath, err := filepath.Abs(possibleConfigPath)
+	if err != nil {
+		return false, err
+	}
+
+	fileExist, err := fileutils.IsFileExists(fullPossiblePath, false)
```

Comment:
Any reason we should use the `IsFileExists` API here, when the `FindFileInDirAndParents` API used below also searches inside the current dir?

Reply:
You are right, and based on our recent changes to the frogbot path (.frogbot/frogbot-config.yml instead of .jfrog/...), we don't need the whole search function.